style.css

```diff
@@ -405,6 +405,79 @@ body {
   cursor: pointer;
 }
 
+/* The Modal (background) */
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.7);
+  overflow: hidden;
+}
+
+/* Modal Content */
+.modal-content {
+  background-color: #fefefe;
+  margin: 3% auto;
+  padding: 20px 60px;
+  border: 1px solid #888;
+  width: 90%;
+  max-width: 700px;
+  backdrop-filter: blur(5px);
+}
```

Comment:
- [ ] Kindly reduce the padding of the popup windows on the mobile version(see image) to follow the Figma design.

### Your Design :point_down: 
![Screenshot from 2023-04-20 17-03-43](https://user-images.githubusercontent.com/91142432/233438887-c34929c8-1bc3-4265-8636-6b904011064f.png)

### Figma's Design :point_down: 
![Screenshot from 2023-04-19 13-30-07](https://user-images.githubusercontent.com/91142432/233438946-8b9f7cd1-87e9-4b79-bf4e-e72868a4fea6.png)